### PR TITLE
Feature/issue 475 redesign wallet process

### PR DESCRIPTION
**Pull request by jgimeno**
Now there is a single process taking new transactions and add them into a queue. This allows to scale the workers.

Closes #

### CODE REVIEW

File: e2e/cluster/node.go [hunk 1]
```diff
@@ -115,6 +115,56 @@ func RedisSpec(nodeSuffix string) (*NodeSpec, error) {
...
+func TransactionsPublisherSpec(nodeSuffix, nsqdAddr, redisAddr, rpcAddr string) (*NodeSpec, error) {
+	id := NodeID("transactions-publisher-" + nodeSuffix)
+	spec := &NodeSpec{
+		ID:          id,
+		Image:       "kowalatech/transactions_publisher:dev",
+		Cmd:         []string{},
+		Env:         []string{
+			fmt.Sprintf("NSQ_ADDR=%s", nsqdAddr),
+			fmt.Sprintf("REDIS_ADDR=%s", redisAddr),
+			fmt.Sprintf("REDIS_ADDR=%s", redisAddr),
```
Comment (acroca) [inline e2e/cluster/node.go]: Double rainbow!
https://www.youtube.com/watch?v=OQSNhk5ICTI

File: e2e/impl/wallet_backend.go [hunk 2]
```diff
@@ -51,7 +51,34 @@ func (ctx *WalletBackendContext) TheWalletBackendNodeIsRunning() error {
 	}
 	redisAddr := fmt.Sprintf("%v:6379", redisIP)
 
-	transactionsPersistanceSpec, err := cluster.TransactionsPersistanceSpec(ctx.globalCtx.nodeSuffix, rpcAddr, redisAddr)
+	nsqlookupdSpec, err := cluster.NsqlookupdSpec(ctx.globalCtx.nodeSuffix)
```
Comment (acroca) [inline e2e/impl/wallet_backend.go]: I think this `TheWalletBackendNodeIsRunning` function is getting way to long. Would be nice to refactor somehow.

File: notifications/cmd/transactions_persistence/main.go [hunk 3]
```diff
@@ -2,25 +2,26 @@ package main
 
 import (
 	"os"
-	"strconv"
-
-	"github.com/sirupsen/logrus"
+		"github.com/sirupsen/logrus"
```
Comment (acroca) [inline notifications/cmd/transactions_persistence/main.go]: The indentation of the imports is wrong

**Comment by yourheropaul** [comment]
@jgimeno Can you resolve the conflicts so we can merge?

**Comment by jgimeno** [comment]
Yeah

**Pull request merged**